Lib/test/libregrtest/refleak.py

```diff
@@ -5,6 +5,7 @@
 import warnings
 from inspect import isabstract
 from test import support
+from _abc import _reset_caches, _get_dump, _reset_registry
```

Comment:
shouldn't this file not depend on the fact that the C implementation is available?

Reply:
Reference counting is only used in CPython AFAIK, so it is OK to unconditionally assume that C extensions are availbable.